Tests/Unit/Module/DnsServerSettingRule.tests.ps1

```diff
@@ -23,6 +23,33 @@ try
 
                 If any option other than "Errors and warnings" or "All events" is selected, this is a finding.'
             }
+            @{
+                IsExistingRule = $True
+                PropertyName = 'NoRecursion'
+                PropertyValue = '$True'
```

Comment:
Same for this one, just to be consistent.

Reply:
Updated